synapse/models/gov/intl.py

```diff
@@ -5,7 +5,7 @@ class GovIntlModule(s_module.CoreModule):
     def getModelDefs(self):
         modl = {
             'types': (
-                ('iso:oid', ('str', {'regex': '^([0-2])((\.0)|(\.[1-9][0-9]*))*$'}), {
+                ('iso:oid', ('str', {'regex': '^([0-2])((\\.0)|(\\.[1-9][0-9]*))*$'}), {
```

Comment:
Similar to before, isn't this changing the meaning of the regex from "match a dot" to "match a backslash and then any character"?

Reply:
```
>>> so = '^([0-2])((\.0)|(\.[1-9][0-9]*))*$'
>>> sn = '^([0-2])((\\.0)|(\\.[1-9][0-9]*))*$'
>>> so == sn
True
>>> so
'^([0-2])((\\.0)|(\\.[1-9][0-9]*))*$'
>>> sn
'^([0-2])((\\.0)|(\\.[1-9][0-9]*))*$'
```